src/rules/syntax/parser_error.ts

```diff
@@ -21,8 +21,8 @@ export class ParserError extends ABAPRule {
     return "parser_error";
   }
 
-  public getDescription(): string {
-    return "Parser error(Unknown statement)";
+  public getDescription(statement: string, abapVersion: string): string {
+    return "Statament does not exist in ABAP" + abapVersion + ": '" + statement + "' If it does, this is a parser error.";
```

Comment:
typo, plus I think this description is too long to fit into a terminal(the filename is shown in the same line)

Reply:
shortened